packages/components/src/mobile/link-settings/index.native.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * External dependencies
+ */
+import { Platform, Clipboard } from 'react-native';
+/**
+ * WordPress dependencies
+ */
+import { compose, usePrevious } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
+import { isURL, prependHTTP } from '@wordpress/url';
+import { useEffect, useState } from '@wordpress/element';
+import { link, external } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import BottomSheet from '../bottom-sheet';
+import PanelBody from '../../panel/body';
+import TextControl from '../../text-control';
+import ToggleControl from '../../toggle-control';
+import UnsupportedFooterControl from '../../unsupported-footer-control';
+import PanelActions from '../../panel/actions';
+import LinkRelIcon from './link-rel';
+
+import styles from './style.scss';
+
+const NEW_TAB_REL = 'noreferrer noopener';
+
+function LinkSettings( {
+	isVisible,
+	onClose,
+	attributes,
+	setAttributes,
+	onEmptyURL,
+	options,
```

Comment:
I think it's worth documenting these props somehow. It will be much easier to use in future :)